dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -925,6 +925,21 @@ CREATE TABLE `t_ds_task_instance` (
 -- Records of t_ds_task_instance
 -- ----------------------------
 
+-- ----------------------------
+-- Table structure for t_ds_task_instance_context
+-- ----------------------------
+DROP TABLE IF EXISTS `t_ds_task_instance_context`;
+CREATE TABLE `t_ds_task_instance_context` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `task_instance_id` int(11) NOT NULL,
+    `context` text NOT NULL,
+    `context_type` varchar(200) NOT NULL COMMENT 'context typ',
+    `create_time` datetime NOT NULL,
+    `update_time` datetime NOT NULL,
+    PRIMARY KEY (`id`),
+    KEY `task_instance_id` (`task_instance_id`,`context_type`) USING BTREE
```

Comment:
Should be unique key?

Reply:
I think we can guarantee it through code logic.